<launch>
    <!-- 定义命令行参数 -->
    <arg name="file" default="file_path" />
    <arg name="interval_sec" default="2" />
    <node pkg="teach_pendant" type="teach_pendant.py" name="teach_pendant_node" args=" --play_from_file $(arg file) --play_file_interval_sec $(arg interval_sec)" output="screen">
    </node>
</launch>